stuff/cron/plagiarism_detector.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+
+
+''' Main Plagiairism Detector Script. 
+
+This script gets all the contest that finised in the last 15 minutes and acesses those contests and gets
+runs copydetect on submissions recieved. 
+
+Finally it pushes the necessary data to the database
+
+'''
+
+import argparse
+import calendar
+import collections
+import datetime
+import json
+import logging
+import operator
+import os
+import shutil
+import sys
+import tempfile
+from typing import (Any, DefaultDict, Dict, Mapping, NamedTuple, Optional,
+                    Sequence, Set, Tuple, Callable, Iterable)
+
+import boto3  # type: ignore
```

Comment:
```suggestion
import boto3  # type: ignore
import copydetect  # type: ignore
```

we should put all the third-party imports together.